charts/gardener/controlplane/charts/runtime/templates/admission-controller/vpa.yaml

```diff
@@ -11,4 +11,11 @@ spec:
     name: gardener-admission-controller
   updatePolicy:
     updateMode: Auto
+  {{- if .Values.global.admission.resources.requests }}
+  resourcePolicy:
+    containerPolicies:
+      - containerName: '*'
+        minAllowed:
+{{ toYaml .Values.global.admission.resources.requests | indent 10 }}
```

Comment:
Can we use separate values for `minAllowed`? I think, you wouldn't use the same values for requests and lower bounds in some cases and the double meaning is confusing.
I would imagine the values structure to look something like this:
```yaml
resources:
  # unchanged ...
  requests:
    cpu: 50m
    memory: 50Mi
  limits:
    cpu: 300m
    memory: 256Mi
vpa:
  enabled: true # breaking change, but I think it makes sense
  minAllowed:
    cpu: 10m
    memory: 25Mi
```

Reply:
Yes that would make more sense. But generally I would move away from landscape dependent configuration for `minAllowed` as discussed in the comment below.

Reply:
@timebertt  As I now use hard-coded min values can you recheck the PR? thanks